src/index.html

```diff
@@ -7,5 +7,18 @@
   </head>
   <body>
     <div id="root"></div>
+    <!--Start of Tawk.to Script-->
+    <script type="text/javascript">
+      var Tawk_API=Tawk_API||{}, Tawk_LoadStart=new Date();
+      (function(){
+      var s1=document.createElement("script"),s0=document.getElementsByTagName("script")[0];
+      s1.async=true;
+      s1.src='https://embed.tawk.to/5ddd210043be710e1d1f2d15/default';
+      s1.charset='UTF-8';
+      s1.setAttribute('crossorigin','*');
+      s0.parentNode.insertBefore(s1,s0);
+      })();
+      </script>
+      <!--End of Tawk.to Script-->
   </body>
 </html>
```

Comment:
Intresting

Reply:
This is a very good suggestion, but in our case I would like to leave it as it is, so we could show this functionality both on the localhost and on the production (just in case something brakes on the prod)

Reply:
Just for your information, there is such option to handle this with `webpack`